docs/pages/database-access/guides/dynamodb.mdx

```diff
@@ -0,0 +1,298 @@
+
+Now, log into the previously configured AWS DynamoDB app on your desktop:
+
+```code
+$ tsh app login --aws-role ExampleTeleportDynamoDBRole aws-dynamodb
+Logged into AWS app aws. Example AWS CLI command:
+
+$ tsh aws s3 ls
+```
+
+The `--aws-role` flag allows you to specify the AWS IAM role to assume when
+accessing AWS API. You can either provide a role name like `--aws-role
+ExampleTeleportDynamoDBRole` or a full role ARN
+`arn:aws:iam::123456789000:role/ExampleTeleportDynamoDBRole`
+
+Now you can use the `tsh aws` command like the native `aws` command-line tool:
+
+```code
+$ tsh aws dynamodb list-tables
+```
+
+To log out of the aws application and remove credentials:
+
+```code
+$ tsh app logout aws-dynamodb
+```
+
+### Using other DynamoDB applications
+
+First, log into the previously configured AWS DynamoDB app if you haven't
+already done so.
+
+```code
+$ tsh app login --aws-role ExampleTeleportDynamoDBRole aws-dynamodb
+```
+
+To connect your DynamoDB application, you can start either a local HTTPS proxy
+or a local AWS Service Endpoint proxy.
+
+<Tabs>
+  <TabItem label="HTTPS proxy">
+  By default, starting the AWS app proxy creates a local HTTPS proxy server
+  that forwards received AWS requests to Teleport backend for AWS application
+  access.
+
+  Now, use the following command to start the proxy your applications will be
+  connecting to:
+
+  ```code
+  $ tsh proxy aws -p 23456
+  Started AWS proxy on http://127.0.0.1:23456.
+
+  Use the following credentials and HTTPS proxy setting to connect to the proxy:
+    AWS_ACCESS_KEY_ID=8cd7ce58b871425e2e05
+    AWS_SECRET_ACCESS_KEY=0e08deb6ceb7538754567286922e7ada75021c40
+    AWS_CA_BUNDLE=/Users/sampleuser/.tsh/keys/teleport.example.com/alice-app/teleport.example.com/aws-dynamodb-localca.pem
+    HTTPS_PROXY=http://127.0.0.1:23456
+  ```
+
+  Use the displayed AWS credentials and HTTPS proxy settings when configuring
+  your application.
+
+  For example, you can configure the AWS credentials and the HTTPS proxy as
+  environment variables for AWS SDK for Python:
+  ```code
+  $ export AWS_ACCESS_KEY_ID=8cd7ce58b871425e2e05
+  $ export AWS_SECRET_ACCESS_KEY=0e08deb6ceb7538754567286922e7ada75021c40
+  $ export AWS_CA_BUNDLE=/Users/sampleuser/.tsh/keys/teleport.example.com/alice-app/teleport.example.com/aws-dynamodb-localca.pem
+  $ export HTTPS_PROXY=http://127.0.0.1:23456
+  $ python3
+  >>> import boto3
+  >>> boto3.client('dynamodb').list_tables()
+  {'TableNames': ['my-dynamodb-table'], 'ResponseMetadata': {...}}
+
+  ```
+
+  </TabItem>
+  <TabItem label="AWS Service Endpoint proxy">
+  If your application cannot use a HTTPS proxy, start the AWS app proxy with
+  the `--endpoint-url` flag to create a local server that can be used as an [AWS
+  Service Endpoint](https://docs.aws.amazon.com/general/latest/gr/rande.html):
```

Comment:
I think we should move this link to the "Next steps" section to avoid breaking the reader's flow